planner/core/prepare_test.go

```diff
@@ -51,6 +51,106 @@ type testPrepareSuite struct {
 type testPrepareSerialSuite struct {
 }
 
+func (s *testPrepareSerialSuite) TestFlushPlanCache(c *C) {
```

Comment:
Please add one more test case about `flush plan-cache` when this feature is not enabled.